scripts/genproto.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Generate rkt protobuf bindings.
+# Run from repository root.
+#
+set -e
+
+if ! [[ "$0" =~ "scripts/genproto.sh" ]]; then
+	echo "must be run from repository root"
+	exit 255
+fi
+
+if ! [[ $(protoc --version) =~ "3.0.0" ]]; then
+	echo "could not find protoc 3.0.0, is it installed + in PATH?"
+	exit 255
+fi
+
+export GOPATH=$(mktemp -d)
+export PATH=${GOPATH}/bin:${PATH}
+
+trap 'rm -rf "${GOPATH}"' EXIT
+
+# git (sha) version of golang/protobuf
+GO_PROTOBUF_SHA="dda510ac0fd43b39770f22ac6260eb91d377bce3"
+
+echo "installing golang/protobuf using GOPATH=${GOPATH}"
+go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
```

Comment:
I am inclined even not to create an issue for that since it becomes more and more obvious to check in the generated code.

Yes this script is a mere helper to have a reproducible way of regenerating the code by pinning the version of golang/protobuf.